apigen/src/main/java/com/distelli/graphql/apigen/ApiGen.java

```diff
@@ -31,12 +31,14 @@
     private Map<String, TypeEntry> generatedTypes = new LinkedHashMap<>();
     private Map<String, TypeEntry> referenceTypes = new HashMap<>();
     private List<TypeEntry> schemaDefinitions = new ArrayList<>();
+    private boolean isAsync = false;
 
     public static class Builder {
         private Path outputDirectory;
         private STGroup stGroup;
         private String guiceModuleName;
         private String defaultPackageName;
+        private String groupFileName = "graphql-apigen.stg";
```

Comment:
Please revert all changes to this file and incorporate the graphql-apigen-async.stg file into graphql-apigen.stg.